superset-frontend/plugins/plugin-chart-echarts/src/Gauge/transformProps.ts

```diff
@@ -146,7 +147,12 @@ export default function transformProps(
   const transformedData: GaugeDataItemOption[] = data.map(
     (data_point, index) => {
       const name = groupbyLabels
-        .map(column => `${column}: ${data_point[column]}`)
+        .map(
+          column =>
+            `${verboseMap ? verboseMap[column] : column}: ${
+              data_point[column]
+            }`,
+        )
```

Comment:
I don't believe this will work if you have a column that doesn't have a verbose name defined for it. I believe you need to do something like this:
```suggestion
        .map(
          column =>
            `${verboseMap[column] || column}: ${data_point[column]}`,
        )
```